control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -321,6 +322,11 @@ func TestRun_ACLPolicyUpdates(t *testing.T) {
 			require.NoError(t, err)
 
 			// Check that the expected policies were created.
+			// There will be more policies returned in the List API that are defaults
+			// existing in Consul on startup, including but not limited to:
+			// * global-management
+			// * builtin/global-read-only
+			// * agent-token
```

Comment:
I'm not sure if it makes sense to backport this entire PR to 1.2.x or not, but I have been running into a failing test there as well for this exact reason. Thought I'd mention it and let you decide. For my own backport PR, I've stolen your changes to get this test passing again.

Reply:
Running into more and more failures -- I'm thinking it does make sense to backport this PR to 1.2.x. Thoughts?

Reply:
@nathancoleman we'd need to cut out the v2 tests fixed here (they don't exist in 1.2.x), but backporting the common fixes through n-2 sounds like the right thing to do.

I'll set a reminder for us to revisit Monday unless you already have it covered.

Reply:
For future readers, see linked PRs below for manual backports into `release/1.1.x` and `release/1.2.x`